captum/insights/api.py

```diff
@@ -32,6 +32,34 @@ def __init__(
         labels: Optional[Tensor],
         additional_args=None,
     ):
+        r"""
+        Constructs batch of inputs to be attributed and visualized.
+
+        Args:
+
+            inputs (tensor or tuple of tensors): Batch of inputs for a model.
+                        These may be either a Tensor or tuple of tensors. Each tensor
+                        must correspond to a feature for AttributionVisualizer, and
+                        the corresponding input transform function of the feature
+                        is applied to each input tensor prior to passing it to the
+                        model. It is assumed that the first dimension of each
+                        input tensor corresponds to the number of examples
+                        (batch size) and is aligned for all input tensors.
+            labels (tensor): Tensor containing correct labels for input examples.
+                        This must be a 1D tensor with length matching the first
```

Comment:
Similar here, this could also just be a list of ints? We'd need to update the type hints as well here.

Reply:
here, label can be a list.

Reply:
Sounds good, let's make this update separately, updating the type hint and docstring.